README.md

```diff
@@ -2,7 +2,7 @@
 
 [![Go Report Card](https://goreportcard.com/badge/storj.io/storj)](https://goreportcard.com/report/storj.io/storj)
 [![Go Doc](https://img.shields.io/badge/godoc-reference-blue.svg?style=flat-square)](https://godoc.org/storj.io/storj)
-[![Coverage Status](https://img.shields.io/badge/coverage-master-green.svg)](https://build.ops.storj.io/job/storj/job/master/Coverage_20Details/)
+[![Coverage Status](https://img.shields.io/badge/coverage-master-green.svg)](https://build.dev.storj.io/job/storj/job/master/781/cobertura/)
```

Comment:
Shouldn't this link be `https://build.dev.storj.io/job/storj/job/master/cobertura`, so that it always opens the latest?

Reply:
True :facepalm: